src/app/[locale]/booking/page.tsx

```diff
@@ -3,6 +3,7 @@ import styles from "./booking.module.css";
 import { useLocale, useTranslations } from "next-intl";
 import { Locale } from "@/i18n/config.types";
 import { Translations } from "@/app/components/Stepper/Stepper.types";
+import { steps } from "@/app/components/Stepper/steps";
```

Comment:
Lets simplify it and just define steps in HomePage which, actually, should be BookingPage)

Reply:
Page renamed, steps moved inside BookingPage component